### PR TITLE
a human readable description for governance proposals added + unit te…

### LINKED ISSUE #1600: Users should be able to reference a human readable description for governance proposals
### Expected Behavior

Governance proposals can be pointed to a human readable description of the proposal written by the proposer. Other community members can leave comments.

### Current Behavior

Only description of the proposal is the proposal txs itself.

### PR DESCRIPTION
**Pull request by Holymir**
### Description

Users should be able to reference a human readable description for governance proposals

### Other changes

governance unit test updated with the new feature

### Related issues

- Fixes #1600

### CODE REVIEW

File: packages/protocol/contracts/governance/Proposals.sol [hunk 1]
```diff
@@ -84,6 +85,11 @@ library Proposals {
     }
   }
 
+  function setDescriptionUrl(Proposal storage proposal, string memory descriptionUrl) public {
```
Comment (asaj) [inline packages/protocol/contracts/governance/Proposals.sol]: Internal?

**Review by asaj — APPROVED** [review]

**Comment by asaj** [comment]
Looks like some tests are failing

File: packages/protocol/contracts/governance/Proposals.sol (hunk 1 above)
Comment (BatiGencho) [reply packages/protocol/contracts/governance/Proposals.sol]: Thx for the advice, I fixed that.

**Comment by BatiGencho** [comment]
> Looks like some tests are failing

@mcortesi There is a single test failing in the group of #executeHotfix() / "should not be executable again", but I guess that is related to the re-executable issue which is already alleviated ?

**Review by mcortesi — APPROVED** [review]

**Comment by mcortesi** [comment]
> > Looks like some tests are failing
> 
> @mcortesi There is a single test failing in the group of #executeHotfix() / "should not be executable again", but I guess that is related to the re-executable issue which is already alleviated ?

From CI it look that the only one failing is the e2e  geth-sync tests...which is probably flaky...

**Pull request merged**